scraper/doc/practicalplants_scraper.md

```diff
@@ -0,0 +1,62 @@
+# Scraper for PracticalPlants
+
+PracticalPlants are not officially maintained since 2013, however, there is a dump of the entire webpage available in our [cloned repository](https://github.com/ElektraInitiative/practicalplants).
+The scraper scrapes the data from the PracticalPlants wiki dump and stores it in `csv` format in the `data` directory.
+
+## Installation and Usage
+
+1. Install dependencies
+
+```shell
+npm install && mkdir -p data
+```
+
+2. Create .env file from .env.example and fill in the required values:
+
+```shell
+cp .env.example .env
+```
+
+`PRACTICALPLANTSPATH` mentioned in .env is the path on your local filesystem to the PracticalPlants wiki dump, which could be fetched from our [repository](https://github.com/ElektraInitiative/practicalplants).
+This is required to scrape the data.
+
+3. Start the scraper
+
+The scraper scrapes the data from the PracticalPlants wiki dump and stores it in `csv` format in the `data` directory. This can be done with the following command:
+
+```shell
+npm run start
+```
+
+As an additional step, the script will also look up the german common name through Wikidata API and store it under the column `Column Name DE` in the `detail.csv` file.
+
+There will be several CSV files generated in the `data` directory:
+
+- `detail.csv`: This file contains the data scraped from the PracticalPlants wiki dump.
+- `errors.csv`: This file contains the errors encountered during the scraping process.
+  The columns in the file identify the type of error i.e. which part of the data was missing.
+  The main error type occurs under the column "Plant Datatable" and represents listing pages e.g. [/abelia][https://practicalplants.org/wiki/abelia/].
+
+4. Correct data manually and merge with generated data (optional)
+
+The scraped data contains inconsistencies and mistakes made by PracticalPlants users.
+In order to correct these mistakes, we need to manually correct the data i.e. copy the `detail.csv` file into a new file.
+The corrected data in the new file should be stored in the same format as the generated data i.e. columns may not be changed.
+
+Since the scraper process could be changed in the future, we should guarantee that the data will not be lost when the scraper is run again.
+Therefore, we should merge the corrected data with the generated data.
+
+Please note the following points when correcting the data:
+
+- The `Binomial name` column should not be changed.
```

Comment:
I kept these namings on purpose because I store all the column names as they are during the fetch step from origin sources, and modify/merge them later during the merge step. However, the good catch here, I should update the documentation stating this